MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include LICENSE
 
 graft websockets/py35
+include websockets/cext.c
```

Comment:
To wrap this up:
- C source files are automatically packaged, no need to declare them in MANIFEST.in
- indeed, there's only so many ways to write this; we all ended up with pretty much the same solution